MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -28,6 +28,9 @@ public class DistanceFormatter: LengthFormatter {
     
     let nonFractionalLengthFormatter = LengthFormatter()
     
+    /// Indicates the most recently used unit
+    public private(set) var unit: LengthFormatter.Unit = .millimeter
```

Comment:
Is this being elsewhere that warrants this change?

Reply:
Yep, it's being used here:

https://github.com/mapbox/mapbox-navigation-ios/blob/95eb404200b64d3d9cd4904e87eca113ffe8970d/MapboxNavigation/InstructionsBannerView.swift#L24